src/utils/addTokenToMetamask.js

```diff
@@ -0,0 +1,49 @@
+import { ChainId, SupportedNetwork, NETWORK_DETAIL } from "../constants";
+
+export async function addTokenToMetamask(ethereum, id, symbol, network) {
+  if (+ethereum.networkVersion !== ChainId[network]) {
+    try {
+      await ethereum.request({
+        method: "wallet_addEthereumChain",
+        params: [
+          { ...NETWORK_DETAIL[ChainId[network]], metamaskAddable: undefined },
+        ],
+      });
+    } catch (err) {
+      console.error(`error adding network to metamask`, err);
+    } finally {
+      return;
+    }
+  }
+
+  const response = await fetch(
+    network === SupportedNetwork.XDAI
```

Comment:
Can we use `if-clauses so it's easier to Arbitrum later on? Like a more readable way than ternary operators.

```
let tokenListURL;
if (network === SupportedNetwork.XDAI) {
    tokenListURL = "https://tokens.honeyswap.org"
}
else if (network === SupportedNetwork.MATIC) {
    tokenListURL = "https://unpkg.com/quickswap-default-token-list@latest/build/quickswap-default.tokenlist.json"
}
```

Reply:
Sure, I'm on it